packages/jss/tests/jss-tests.ts

```diff
@@ -72,6 +72,31 @@ attachedStyleSheet.addRules({
   }
 })
 
+// fallbacks test
+attachedStyleSheet.addRules({
+  rule4: {
+    fallbacks: {
```

Comment:
there is one more syntax that is supported that you might want to test - array 

https://cssinjs.org/jss-syntax#fallbacks

Reply:
```js
const styles = {
  container: {
    display: 'flex',
    fallbacks: [{display: 'box'}, {display: 'flex-box'}]
  }
}
```